.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{.Values.todoapp.RBAC.name}}
```

Comment:
Ensure that `{{.Values.todoapp.RBAC.name}}` is defined in your `values.yaml` file for the `todoapp`. If it's not defined, it could lead to RBAC configuration issues.